src/flaskwebgui.py

```diff
@@ -45,13 +45,16 @@ def find_browser_on_linux():
             return path
 
     for path in paths:
-        bp = (
-            subprocess.check_output(["which", path.split("/")[-1]])
-            .decode("utf-8")
-            .strip()
-        )
-        if os.path.exists(bp):
-            return bp
+        try:

```

Comment:
Hi, thank you for your contribution!

Try this instead of try/except block:

```
from contextlib import suppress

with suppress(subprocess.CalledProcessError):
    bp = (
                subprocess.check_output(["which" etc

```